### DISCUSSION TITLE
Understanding question about lyrics in flac files. UNSYNCEDLYRICS flac tag

### DISCUSSION THREAD

**Discussion by trclst** [jcorporation/myMPD#1155] [Q&A] [answered]
https://jcorporation.github.io/myMPD/references/lyrics

The term **configurable vorbis comments in flac** is used there.  What exactly does this mean? The COMMENT tag?
Is the UNSYNCEDLYRICS supported? Because myMPD don't show anything under lyrics tab in song info.

I have flac files here from bandcamp where `ffprobe` outputs the following:
```

Input #0, flac, from 'GUNSHIP - UNICORN - 01 Monster In Paradise (Feat. Milkie Way, Dave Lombardo, Tyler Bates).flac':
  Metadata:
    TITLE           : Monster In Paradise (Feat. Milkie Way, Dave Lombardo, Tyler Bates)
    ARTIST          : GUNSHIP
    DATE            : 2023
    COMMENT         : Visit https://gunshipmusic.bandcamp.com
    ALBUM           : UNICORN
    track           : 1
    album_artist    : GUNSHIP
    ISRC            : USA2P2251728
    UNSYNCEDLYRICS  : All this living is so worth dying for
                    : So come with me
                    : I can make you feel alive
                    : If you are hearing this
                    : You are in paradise
                    : 
```

Thanks in advanced !

**Comment by jcorporation** [comment, marked as answer]
You can configure the vorbis comment names from which myMPD reads synced and unsynced lyrics. Enable it in settings -> features and you can configure the comment names. Defaults is LYRICS for unsynced lyrics.

**Reply by trclst** [reply]
Thank you ! i replace **LYRICS** with **UNSYNCEDLYRICS** and the lyrics from flac vorbis tag show up but one thing i notice is that the lyrics of every track begin with "=" what does not seem normal if i look at ffprobe output.

![lyrics](https://github.com/jcorporation/myMPD/assets/141440559/b1a0cd8f-ce6f-415a-9415-146687113a2a)
